### PR TITLE
Use a shared 'tool' for all curator actions and use blacklight config for tools + actions

### PR DESCRIPTION
**Pull request by cbeer**
Screen Shot 2021-03-03 at 10 11 02

### CODE REVIEW

**Comment by mejackreed** [comment]
I think we do want this, but the ticket was created upstream in Blacklight

**Comment by ggeisler** [comment]
Yep, that is definitely the end result we want, but whether it makes sense to do it here or in Blacklight I'll leave up to you all.

One thing I see from the screenshot @cbeer is that the existing buttons on the bottom row seem to have lost their horizontal spacing (between Sort and Per page and between Per page and the result view buttons). There wasn't a lot of horizontal space before but there was some, and from the screenshot it appears there is none.

**Comment by cbeer** [comment]
thanks; I'm not sure where that spacing magically comes from, but i've added some back.

File: app/controllers/concerns/spotlight/catalog.rb [hunk 1]
```diff
@@ -27,12 +27,7 @@ def add_facet_visibility_field
                                         }
     end
 
-    def render_save_this_search?
-      (current_exhibit && can?(:curate, current_exhibit)) &&
-        !(params[:controller] == 'spotlight/catalog' && params[:action] == 'admin')
-    end
-
-    def render_bulk_actions?
+    def render_curator_actions?
```
Comment (mejackreed) [inline app/controllers/concerns/spotlight/catalog.rb:30]: I'm good with this, but @ggeisler is there a reason on when we would want to show "Save search" and "Bulk actions" separately? Does this make #2685 impossible?

**Review by mejackreed — COMMENTED** [review]
This looks great to me, just had one question about us enabling bulk actions in the "Browse" context.

File: app/controllers/concerns/spotlight/catalog.rb (hunk 1 above)
Comment (cbeer) [reply app/controllers/concerns/spotlight/catalog.rb:30]: We should be able to put `if`/`unless` conditions on the individual tools.

Comment (mejackreed) [reply app/controllers/concerns/spotlight/catalog.rb:30]: WFM

**Review by mejackreed — APPROVED** [review]

**Pull request merged**